packages/babel-plugin-external-helpers/src/index.js

```diff
@@ -1,9 +1,12 @@
+import { declare } from "@babel/helper-plugin-utils";
 import { types as t } from "@babel/core";
 
-export default function() {
+export default declare(api => {
+  api.assertVersion(7);
```

Comment:
Definitely curious for feedback on this approach. It's a little verbose, but it means that since this is the function that throws, the preset/plugin doing the validation is the thing that throws, so the name of it shows up in the stack trace making debugging a lot nicer.

Reply:
Would it make sense to convert this to "assertSemverVersioning" or something like that? Presumably it will be possible for plugins to be compatible with multiple major versions of Babel.

Reply:
Hmm, I worry about that getting pretty verbose. Is the concern that people will assume that `assertVersion` requires a single version?

`api.assertVersion(">=7.x < 9.x")` I at least think is acceptable.

Reply:
>  Is the concern that people will assume that assertVersion requires a single version?

Yeah exactly. When I read `assertVersion` and saw your example it wasn't immediately obvious that a semver range could be given.

Reply:
> Is the concern that people will assume that assertVersion requires a single version? Yeah exactly. When I read `assertVersion` and saw your example it wasn't immediately obvious that a semver range could be given.

Reply:
I feel like documentation would probably be enough to address that, but I do see where you're coming from. I was thinking of the function like `assertVersionSatisfies(...)` so the `version` is the Babel core version, not argument string, so the fact that the string can be semver seemed not strongly tied to the mention of version in the name. 

At the end of the day, most people  bothering to put this assertion in in the first place will probably look at our docs, or else copy paste and not give it a second thought. Since this is only an issue once we start releasing future versions, it seems expected that once it becomes important, people will wonder what specifically it supports as an argument, and at that point the docs are enough.